client/src/components/Header/Header.js

```diff
@@ -32,6 +42,11 @@ const MenuButton = styled.button`
   border: none;
   position: absolute;
   left: 2 rem;
+  img {
+    position: ${({ open }) => (open ? "fixed" : "relative")};
+    fill: ${({ open }) => (open ? "#383636" : "white")};
```

Comment:
If you can, try sticking to one convention of naming your colors 
```suggestion
    fill: ${({ open }) => (open ? "#383636" : "#fff")};
```